alica_engine/src/engine/teammanager/TeamManager.cpp

```diff
@@ -3,11 +3,13 @@
 #include "engine/AlicaEngine.h"
```

Comment:
You can remove the engine include and add the following includes instead:

#include "engine/IAlicaCommunication.h"
#include "engine/ConfigChangeListener.h"
#include "engine/PlanRepository.h"